.mergify.yml

```diff
@@ -168,6 +168,21 @@ pull_request_rules:
           into master and ride the normal stabilization schedule. Exceptions
           include CI/metrics changes, CLI improvements and documentation
           updates on a case by case basis.
+  - name: Reminder to update RPC clients for changes in `rpc/`
+    conditions:
+      - or:
+        - files~=^rpc/src/rpc\.rs$
+        - files~=^rpc/src/rpc_pubsub\.rs$
+        - files~=^rpc-client-api/src/.*\.rs$
```

Comment:
I _think_ this about covers the files that could possibly have an impact on the public RPC API. If you think that we should just dragnet every `.rs` file in `rpc/` and `rpc-client-api/` then I can do that too!

Reply:
This seems like a good place to start